### PR TITLE
Auto OCP-41132 OCP-41179

### PR DESCRIPTION
**Pull request by huiran0826**
Auto OCP-41132 OCP-41179

Logs:
https://privatebin-it-iso.int.open.paas.redhat.com/?748c12a2010d55a1#23GZVV2RSSVsWMN9Us3SJ4Cxmfokj9GgyTPXkb8WMk17
https://privatebin-it-iso.int.open.paas.redhat.com/?9b1b8a78f8f6e10b#BDZzPtVW6ta8Fpqkyi2oQPSv4s9FzcQ8CUfP3B3qvuey

Could you help take a look? Thanks!
@rbbratta @weliang1 @zhaozhanqi  @asood-rh @kakkoyun @brahaney

### CODE REVIEW

File: testdata/networking/ovn-egressfirewall/egressfirewall-policy5.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    to:
+      dnsName: api.segment.io
+  - type: Allow
+    to:
+      dnsName: login.ibm.com
+  - type: Allow
+    to:
+      dnsName: marketscan2.truvenhealth.com
+  - type: Allow
+    to:
```
Comment (zhaozhanqi) [inline testdata/networking/ovn-egressfirewall/egressfirewall-policy5.yaml:65]: Above Allow dnsName is needed for this testing? saw case only use 'www.amihealthy.com'

Comment (zhaozhanqi) [inline testdata/networking/ovn-egressfirewall/egressfirewall-policy5.yaml:65]: Above Allow dnsName is needed for this testing? saw case only use 'www.amihealthy.com'

Comment (huiran0826) [reply testdata/networking/ovn-egressfirewall/egressfirewall-policy5.yaml:65]: In the original issue, only first 5 dnsName rules were applied, and no status displaying when "oc get egressfirewall".  I don't think we need to check each dnsName for the curl result, so here just check egressfirewall status to make sure the egressfirewall policy was applied and the last dnsName curl result was correct.

Comment (huiran0826) [reply testdata/networking/ovn-egressfirewall/egressfirewall-policy5.yaml:65]: Same as above.

**Pull request merged**